airbyte-webapp/src/scss/_z-indices.scss

```diff
@@ -1,2 +1,4 @@
 $sidebar: 9999;
-$tooltip: 9999 + 1;
+$tooltip: 9999 + 2;
+$modal: 9999 + 1;
+$panelSplitter: 0;
```

Comment:
:art: Small nitpick, but could we try to keep this file in order with the actual z-indices (either descending or ascending, so that you can get the order of components directly from the variable order in this file.